fiatconnect-api.md

```diff
@@ -2227,12 +2228,56 @@ A KYC schema containing personal data about a user, as well as documents such as
 	},
 	phoneNumber: `string`,
 	selfieDocument: `string`,
-	identificationDocument: `string`
+    identificationDocument: `string`
 }
 ```
 
 The `selfieDocument` and `identificationDocument` fields should be base64 encoded binary blobs representing images.
 
+#### 9.3.1.2. `PersonalDataAndDocumentsDetailed`
+A more detailed Schema allowing the user to provide more information to give them a greater chance to be admitted.
```

Comment:
```suggestion
A more detailed Schema allowing the user to provide more information to work with CICO providers that require it.
```
Seems like "to give them a greater chance to be admitted" is misleading-- sounds like the user can optionally provide more info, whereas some providers may *only* accept KYC attempts with this extra set of information